contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinLido.sol

```diff
@@ -59,12 +86,43 @@ contract MixinLido {
         internal
         returns (uint256 boughtAmount)
     {
-        (ILido lido) = abi.decode(bridgeData, (ILido));
-        if (address(sellToken) == address(WETH) && address(buyToken) == address(lido)) {
+        if (address(sellToken) == address(WETH)) {
+            return _tradeStETH(buyToken, sellAmount, bridgeData);
+        } 
+
+        return _tradeWstETH(sellToken, buyToken, sellAmount, bridgeData);
+    }
+
+    function _tradeStETH(
+        IERC20TokenV06 buyToken,
+        uint256 sellAmount,
+        bytes memory bridgeData
+    ) private returns (uint256 boughtAmount) {
+        (IStETH stETH) = abi.decode(bridgeData, (IStETH));
```

Comment:
@dekz I tried to make the contract backward compatible. 

With the change in PR `bridgeData` will always contain two addresses (stETH and wstETH) but I think that's fine.